azure-mgmt-msi/README.rst

```diff
@@ -0,0 +1,49 @@
+Microsoft Azure SDK for Python
+==============================
+
+This is the Microsoft Azure MSI Management Client Library.
+
+Azure Resource Manager (ARM) is the next generation of management APIs that
```

Comment:
Is this the right description for MSI?

Reply:
That's correct I feel, what would you do?